spec/services/trainees/create_from_hesa_spec.rb

```diff
@@ -35,6 +38,8 @@ module Trainees
       allow(Sentry).to receive(:capture_message)
       create(:nationality, name: nationality_name)
       create(:provider, ukprn: student_attributes[:ukprn])
+      create(:provider, ukprn: accredited_provider_ukprn)
+      create(:lead_partner, :hei, urn: lead_partner_urn)
```

Comment:
We haven't added `urn`s for HEI lead partners so this doesn't really reflect reality. Can we use UKPRN here?